force-app/main/default/classes/RD2_ERecurringDonationsSelector.cls

```diff
@@ -35,6 +35,10 @@
 */
 public inherited sharing class RD2_ERecurringDonationsSelector {
 
+	private final String ALL_RD = 'All Recurring Donations';
+	private final String ELEVATE_RD = 'Elevate Recurring Donations';
+	private final String NONELEVATE_RD = 'Non-Elevate Recurring Donations';
```

Comment:
These look like hardcoded picklist value strings. Where are these defined for use elsewhere?

Reply:
Hello Michael, these values were defined [here](https://salesforce.quip.com/kRYaAjEE8aHr?range_cols=1&range_rows=4#temp:s:temp:C:PQJ39e645cd36ea4f44919fc2850;temp:C:PQJf83d97a2a7bc4183b0315cc97). They are used [here](https://github.com/SalesforceFoundation/NPSP/blob/feature/240__filter_rd_elevate_nonelevate/force-app/main/default/lwc/rd2RecurringDonation/rd2RecurringDonation.js-meta.xml) To show this options on the Experience Builder, and then only show the Recurring Donation records based on this filter.
Please let me know if you need help here.

Reply:
Thanks. I see they're related to text in the LWC component config. I was trying to check on whether those can ever be translated, but it doesn't look like that's the case. So, these are probably fine - at least for now.